docker/local-development.md

```diff
@@ -25,19 +25,23 @@ tutorial that can help connect the dots between these concepts. Give it a whirl
 
 ## Overview
 
-A containerized local development environment has four components, and one
+A containerized local development environment has three components, and one
 optional service:
```

Comment:
I think this should say:

> four components, and ~one~**two** optional service**s**:

This PR seems to me to have shown this sentence to be difficult to maintain, however, so maybe we should just replace it with something like:

> A containerized local development environment has a few components, including some optional services: